lib/danger/danger_core/plugins/dangerfile_github_plugin.rb

```diff
@@ -92,6 +92,14 @@ def self.instance_name
       "github"
     end
 
+    # @!group PR Review
+    # Current PR Review object.
+    # @return [GitHubReview]
+    #
```

Comment:
> @return [GitHubReview] -> @return [GitHubSource::Review] or we can just leave Review.

IMO `@return Review` is fine, maybe even `@return ReviewDSL` as these docs are for end-users who don't care about our internal semantics

> Should we mention github.review.markdown(...) ?

Yep :D - please make it a real-ish example like the ones above